lib/flutter_typeahead.dart

```diff
@@ -705,6 +709,25 @@ class _TypeAheadFieldState<T> extends State<TypeAheadField<T>>
       if (this._effectiveFocusNode.hasFocus) {
         this._suggestionsBoxController.open();
       }
+
+      ScrollableState scrollableState = Scrollable.of(context);
+      if (scrollableState != null) {
+        // The TypeAheadField is inside a scrollable widget
+        scrollableState.position.isScrollingNotifier.addListener(() {
+          bool isScrolling = scrollableState.position.isScrollingNotifier.value;
+          _resizeOnScrollTimer?.cancel();
+          if (isScrolling) {
+            // Scroll started
+            _resizeOnScrollTimer =
+                Timer.periodic(Duration(milliseconds: 500), (timer) {
```

Comment:
Let's make the 500 ms a const variable at the top so we can easily modify it.. and .. honestly 1/2 second seems sort of long?  Maybe 300 ms or 250?  Did you try different values?

Reply:
Alright. Lower values should work fine. I will add it as a const.

Reply:
@sjmcdowall I tried lower values like 200 or 300, but those make the impression of the suggestion box lagging (On a usual scroll to show the suggestions, a low value maybe resizes the box 2 or 3 times, looking like it is drawing at low fps). A value like 500 ms leaves some room for the user to scroll and then resize only once, looking nicer.
What is your opinion?

Reply:
OK - lets try 500 and see how it goes..